js/components/portal-dashboard/response-details/response-details.tsx

```diff
@@ -54,14 +60,18 @@ export class ResponseDetails extends React.PureComponent<IProps, IState> {
     this.state = {
       selectedStudents: [],
       showSpotlightDialog: false,
-      showSpotlightListDialog: false
+      showSpotlightListDialog: false,
+      feedbackLevel: "Question"
     };
   }
+
   render() {
     const { activities, anonymous, answers, currentActivity, currentStudentId, currentQuestion, hasTeacherEdition, isAnonymous, listViewMode, questions,
       setAnonymous, setCurrentActivity, setCurrentQuestion, setCurrentStudent, setListViewMode, setStudentFilter, sortByMethod, sortedQuestionIds, studentCount, students,
-      trackEvent } = this.props;
-    const { selectedStudents, showSpotlightDialog, showSpotlightListDialog } = this.state;
+      trackEvent, viewMode } = this.props;
+
+    const { selectedStudents, showSpotlightDialog, showSpotlightListDialog, feedbackLevel } = this.state;
+
     // TODO: FEEDBACK
     // if feedback is on, show the QuestionFeedbackPanel or the Activity FeedbackPanel
```

Comment:
Is this comment obsolete now?  Can we remove it?  I think it was my old TODO anticipating this work.